### ISSUE TITLE
ImportError for 'google.cloud.speech'

### ISSUE THREAD

**Issue by gitdesai** [GoogleCloudPlatform/google-cloud-python#5251]
Working from google console. I had cloned your project 2 weeks back and I was able to execute quickStart & transcribe.py. However, since couple of days I get an error "from google.cloud import speech". Kindly help.

**Comment by gitdesai** [comment]
Are these steps required if I am using gcp console?
I am trying to execute quickstart.py from https://github.com/GoogleCloudPlatform/python-docs-samples.git/python-docs-samples/speech/cloud-client

**Comment by crwilcox** [comment]
@gitdesai a little bit back a change was made to Cloud Console to only contain the `Production/Stable` Libraries. google-cloud-speech, currently v0.34.0, is currently Beta so you will need to `pip install google-cloud-speech`

**Comment by gitdesai** [comment]
Hmm! thank you for your responses.

I am definitely confused. But may be I have not been able to explain the problem correctly. However, I have been able to get my work done.

I was trying to run the script on "Google Cloud Shell" which comes with pre-installed google cloud SDK and I don't have to / no permissions to install either. So, I was able to get the script working on this one earlier which has now stopped working. May be because of permissions/recent changes.

I created a new compute node and here, I was able to install google cloud and run the script.


Thanks,
Desai.
